We deselected some test so the coverage dropped as well.
https://github.com/domdfcoding/domdf_python_tools/issues/82

--- domdf_python_tools-3.9.0/tox.ini.orig
+++ domdf_python_tools-3.9.0/tox.ini
@@ -220,7 +220,7 @@
 [coverage:run]
 
 [coverage:report]
-fail_under = 95
+fail_under = 92
 show_missing = True
 exclude_lines =
     raise AssertionError
